weldx/transformations/rotation.py

```diff
@@ -63,45 +64,32 @@ def rotation_matrix_z(angle) -> np.ndarray:
     return _Rotation.from_euler("z", angle).as_matrix()
 
 
+@inherit_docstrings
```

Comment:
while it would be nice to have the original docstring here we should just stick to linking the scipy docs since sphinx just goes bananas otherwise 🍌

Reply:
noticed that as well. I guess scipy does not use warnings as errors policy, but just keeps accumulation wrong formattings since the beginning of time...

Reply:
feel free to use https://github.com/marscher/weldx/pull/1 here, it should be enough for now 👍

Reply:
Thanks!